src/app/Settings/NotificationControl.tsx

```diff
@@ -92,7 +115,26 @@ const Component = () => {
     <>
       <Stack hasGutter>
         <StackItem key="all-notifications">
-          <Switch id="all-notifications" label="All Notifications" isChecked={allChecked} onChange={handleCheckAll} />
+          <Level>
```

Comment:
Do you think we should have the number spinner right below the switch? Then, the switch and the spinner, each can have a label (similar to how we labeling formgroup).

Right, if viewing in a large screen, it might be hard recognize there is a spinner on the right side.

![Screenshot from 2022-12-15 16-01-43](https://user-images.githubusercontent.com/68053619/207966340-ae0f4971-4e9e-401a-aef4-037858843130.png)

Reply:
Looking at this, the spinner should allow manual input?